java/Ice/secure/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ZeroC, Inc.
+
+plugins {
+    // Register the Ice Builder plugin without applying it here.
+    // The plugin will be applied to all subprojects in the `subprojects` block below.
+    id("com.zeroc.ice.slice-tools") version "3.8.0-+" apply false
+}
+
+subprojects {
+    // Apply the application plugin to enable Java application support in subprojects.
+    apply(plugin = "application")
+
+    // Apply the Gradle Slice Tools for Ice and Java plugin to enable Slice compilation in all subprojects.
+    apply(plugin = "com.zeroc.ice.slice-tools")
+
+    repositories {
+        // Add the ZeroC Nightly repository for downloading Ice runtime and development artifacts.
```

Comment:
> for downloading Ice runtime and development artifacts.

This is not syntactically correct, and I am not fond of the expression "Ice runtime and development artifacts".

Reply:
How about `// Add the ZeroC Nightly maven repository, required to use the Ice nightly builds.`